types/semver-compare-multi/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for semver-compare-multi 1.0
+// Project: https://github.com/zenithpolar/semver-compare
+// Definitions by: ravenclaw900 <https://github.com/ravenclaw900>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function cmp(a: string, b: string): number;
+export function newComparer(separators?: string[]): (a: string, b: string) => number;
```

Comment:
These could also return `-1 | 0 | 1` to be more specific, but I think that `number` is fine.